### ISSUE TITLE
NodeHasPkgCount.java:35-36: Use this matcher in `RpmTest`...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#311]
The puzzle `307-cda9f9c1` from #307 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/2054c9db6420a9780c6f62209e0b978cf7783d7b/src/test/java/com/artipie/rpm/hm/NodeHasPkgCount.java#L35-L36

The puzzle was created by @olenagerasimova on 30-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8), @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#311](https://github.com/artipie/rpm-adapter/issues/311) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/3015aecf-2669-4f02-a55b-dfedc3584fc9); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@661823d7-f32c-4fcc-bf9e-6e35e0472a13, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Week 27' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.53.15](https://github.com/zerocracy/farm): CID: [b8cde747-8133-4db3-8d6f-45d56a61bb3d](https://www.0crat.com/footprint/CT2E6TK9B/b8cde747-8133-4db3-8d6f-45d56a61bb3d), Type: "Job milestoned"

**Comment by 0pdd** [comment]
The puzzle `307-cda9f9c1` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317) are still not solved.

**Comment by 0crat** [comment]
Job was finished in 28 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#359](https://github.com/artipie/rpm-adapter/issues/359), [#360](https://github.com/artipie/rpm-adapter/issues/360) are still not solved; solved: [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#360](https://github.com/artipie/rpm-adapter/issues/360) is still not solved; solved: [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317), [#359](https://github.com/artipie/rpm-adapter/issues/359).
